accounts/account.md

```diff
@@ -18,11 +18,11 @@ class Account:
     // verifies the signature using the public key of the account
     verify(data: bytes, signature: bytes) : boolean;
 
-    // serializes the transaction, signs the transaction, applies the signature on the transaction
-    sign_transaction(transaction: Transaction);
+    // serializes the transaction, signs the transaction, returns the signature;
+    sign_transaction(transaction: Transaction): bytes;
 
-    // serializes the message, signs the message, applies the signature on the message
-    sign_message(message: Message);
+    // serializes the message, signs the message, returns the signature;
+    sign_message(message: Message): bytes;
```

Comment:
Might confuse a few developers (signature both applied on _object.signature_ and returned). Is this required?

Reply:
I think the comment was confusing, I meant "computes the signature and returns it". Changed it!